spec/relativetimeformat.html

```diff
@@ -100,8 +102,14 @@ <h1>PartitionRelativeTimePattern (relativeTimeFormat, value, unit)</h1>
           1. Let _tl_ be `"future"`.
         1. Let _po_ be ? Get(_patterns_, _tl_).
         1. Let _fv_ be ! FormatNumber(_relativeTimeFormat_.[[NumberFormat]], _value_).
-        1. Let _pr_ be ! ResolvePlural(_relativeTimeFormat_.[[PluralRules]], _value_).
-        1. Let _pattern_ be ? Get(_po_, _pr_).
+        1. Let _type be _relativeTimeFormat_.[[Type]].
+        1. If _type_ is equal to "text", then
+          1. Let _exists_ be ? HasProperty(_po_, _value_).
+          1. If _exists_ is *true*, then
+            1. Let _pattern_ be ? Get(_po_, _value_).
+        1. If _pattern_ is *undefined*, then
+          1. Let _pr_ be ! ResolvePlural(_relativeTimeFormat_.[[PluralRules]], _value_).
+          1. Let _pattern_ be ? Get(_po_, _pr_).
```

Comment:
I think rather than adding in those extra steps, we actually want to skip steps 4-5 if in numeric mode.

Reply:
Which steps 4-5? When I build the spec with my patch, steps 4-5 are:
```
4. If isFinite(value) is false, then throw a RangeError exception.
5. Let unit be ? SingularRelativeTimeUnit(unit).
```

Is that what you're asking to skip?

Reply:
Sorry, I meant steps 14-15, namely

        1. Let _exists_ be ? HasProperty(_patterns_, ToString(_value_)).
        1. If _exists_ is *true*, then
          1. Let _result_ be Get(_patterns_, ToString(_value_)).
          1. Return a List containing the Record { [[Type]]: `"literal"`, [[Value]]: _result_ }.

Reply:
oh, of course!